### ISSUE TITLE
Add tags to the HSF repos

### ISSUE THREAD

**Issue by amanmdesai** [hsf-training/maintenance#12]
At present there are no tags present in many of the HSF repos. 

Maybe we should consider tagging a repository when it is used for a talk or carpentry?
 
(I am not sure if such a process is already implemented)

**Comment by klieret** [comment]
You mean the "topics" under the description? I agree. I already wrote in Slack looking for more people to help to add some.

**Comment by amanmdesai** [comment]
I meant this one: the tag for a given repo e.g., https://github.com/hsf-training/maintenance/tags

not sure if thats same as what you said.

**Comment by klieret** [comment]
Oh I see, yeah that's different. This could be done. However, unless one also offers different versions of the deployed webpage, I doubt that this would be of much use for the students. 

I do however think that every readme should contain a list of links to the workshops that have taught it.

**Comment by amanmdesai** [comment]
> Oh I see, yeah that's different. This could be done. However, unless one also offers different versions of the deployed webpage, I doubt that this would be of much use for the students.

oh I see. so we cant deploy all the tags at once?   

> I do however think that every readme should contain a list of links to the workshops that have taught it.

thats a good idea. 
For the moment, It would also be good to post the link to the next hackathon in the singularity/docker readme?

**Comment by klieret** [comment]
I think there's ways to get multiple versions with gh pages, but I think it's probably not worth it (?).

Yes, we can also add a list of past workshops/hackathons to the readmes. Feel free to open PRs adding these links.

Later today I will think about what to do to bring the readmes in a standard format.

**Comment by amanmdesai** [comment]
Yes if we add list of workshops/hackathons with links in the readme, then it wont be worth to deploy multiple versions.

**Comment by klieret** [comment]
I will start creating a template in [hsf-training/carpentry-cookiecutter](https://github.com/hsf-training/carpentry-cookiecutter).

Though actually this should also go in the HTML version of the modules....
